### ISSUE TITLE
Virtual box or brackets

### ISSUE THREAD

**Issue by ShadowCodeRz** [adobe/brackets#15166]
I'm using linux mint 19.3 and brackets 1.14.1 and it requires me to choose between having virtual box installed or brackets.

**Comment by roubarbe** [comment]
Virtual Box is a tools to manage Virtual Machines, and Brackets is a text editor.

What are you trying to do, more precisely ?

**Comment by ShadowCodeRz** [comment]
Exactly,  they're completely different apps but for some reason to install one I need to uninstall the other
 #![15906443798421991063162572350485](https://user-images.githubusercontent.com/24922476/83103199-3d223280-a084-11ea-894d-1fdb1664a3d4.jpg)

**Comment by roubarbe** [comment]
Looks related to [this](https://github.com/adobe/brackets/issues/15151#issue-619571301)

**Comment by rodrigosantos-git** [comment]
This happened to me also on Ubuntu 18.04.
